Analysis/ingrid/likelihood.nim

```diff
@@ -1761,6 +1770,18 @@ proc main(
     if mlp.len > 0 or convnet.len > 0:
       raise newException(Exception, "Using neural network vetoes is only supported if the program is compiled " &
         "using the C++ backend!")
+#Using this to test the new implementations, need to be removed at some point        
+    echo "path:"
+    ToAProbabilityHists ="../../resources/ToA_P_densitys.csv"
+    var df = newDataFrame()
+    var Energy_list = newSeq[float]()
+    var test = newDataFrame()
+    df = readToAProbabilitys(ToAProbabilityHists, Energy_list)
+    test = getInterpolatedDfToA(df,Energy_list)
```

Comment:
Just as an explanation: 

You could just as well avoid the `var df / var test` lines and just write `let df = readToA...; let test = getInterp...`.